PubSub/sdk/src/main/java/org/lantern/pubsub/AutoStart.java

```diff
@@ -0,0 +1,19 @@
+package org.lantern.pubsub;
```

Comment:
Yeah given that it's a separate library outside of our Android app that sounds good to me @oxtoacart and @atavism